### DISCUSSION TITLE
Hex editor pane comes up blank

### DISCUSSION THREAD

**Discussion by profbbrown** [eclipse-theia/theia#13519] [Q&A]
I've installed the hex editor extension using the Extensions manager. Whenever I try to open a file using it, the pane is empty. It's using a webview, which from experience causes problems for some Theia installations. But webview works fine for things like image viewing. (I've had to set the webview endpoint to `{{hostname}}` in order to get it to work.)

Attached is a screenshot of the two errors that appear in the JavaScript console.

<img width="751" alt="Screenshot 2024-03-22 at 1 11 59 PM" src="https://github.com/eclipse-theia/theia/assets/1850976/3d79c977-9af2-4629-827f-b7f09949e8e3">

Is there anything else I need to configure for the hex editor to function properly?

**Comment by JonasHelming** [comment]
Which hex editor extension is this exactly and which Theia version are you using?

**Comment by profbbrown** [comment]
I'm using ms-vscode.hexeditor, version 1.9.14, installed from the Extensions pane.

Theia version 1.47.0, in a web browser.

**Comment by JonasHelming** [comment]
OK, thank you. I can reproduce this. Would you mind transforming this into an issue, you can use the GitHub option under the three dots to do this for you.

**Comment by profbbrown** [comment]
Opened as issue #13532
